backend/docs/docs.go

```diff
@@ -360,18 +360,52 @@ const docTemplate = `{
                 }
             }
         },
-        "/spi-webhook": {
+        "/service/available-presets": {
             "get": {
+                "security": [
```

Comment:
This should be public, no?

Reply:
I was thinking about that as well but that would mean that this would be the only endpoint that can be used without being logged in :thinking: 
Therefore it seemed more consistent to make this also protected and I didnt see any downsides for doing so